oidc_apis/scopes.py

```diff
@@ -66,10 +66,10 @@ def scope_ad_groups(self):
         }
 
 
-class CustomInfoTextStandardScopeClaims(StandardScopeClaims):
+class ReducedStandardScopeClaims(StandardScopeClaims):
     info_profile = (
         _('Basic profile'),
-        _('Access to your basic information. Includes names, gender, birthdate and other information.'),
+        _('Access to your basic information. Includes names, nickname and possible user picture.'),
```

Comment:
This message could only mention "names and a possible user picture" as the nickname in userinfo is set as the user's first name in `tunnistamo.oidc.get_userinfo` in the line below (and nickname could be considered to be included in 'names').
https://github.com/City-of-Helsinki/tunnistamo/blob/c4d622f29f6bfaf0269f30892415eabd7226d8c6/tunnistamo/oidc.py#L41